pkg/resources/prometheus/configmap-rules.go

```diff
@@ -344,6 +344,13 @@ groups:
     labels:
       kubermatic: federate
 
+- name: kubermatic.kubernetes.apiserver
```

Comment:
There is a job, `kubermatic.etcd` defined at line 44, let's make the recording rule from here part of `kubermatic.etcd`.